packages/accounts-base/accounts_common.js

```
@@ -297,11 +297,21 @@ export class AccountsCommon {
       }
     }
 
-    if (options.collection && options.collection !== this.users._name && options.collection !== this.users) {
-      this.users = this._initializeCollection(options);
+    if (options.collection) {
+      const isNewCollection =
+        (options.collection instanceof Mongo.Collection)
```

ESLint

[error] Replace `(options.collection·instanceof·Mongo.Collection)` with `options.collection·instanceof·Mongo.Collection`

(prettier/prettier)